src/main/java/mate/academy/bookdao/BookDaoImpl.java

```diff
@@ -0,0 +1,141 @@
+package mate.academy.bookdao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dbconnection.ConnectionUtil;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Book;
+
+@Dao
+public class BookDaoImpl implements BookDao {
+
+    @Override
+    public int getRowsCount() {
+        String sql = "SELECT COUNT(*) AS total_rows FROM book";
```

Comment:
The table name should be `books` instead of `book` according to the SQL style guide in the checklist.